classes/PodsInit.php

```diff
@@ -485,13 +494,17 @@ public function register_assets() {
 		}
 
 		wp_register_script(
-			'pods', PODS_URL . 'ui/js/jquery.pods.js', array(
+			'pods',
+			PODS_URL . 'ui/js/jquery.pods.js',
+			array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 625bc5885b16b1d99b5641aa04231868</sub>